qa/os/windows-2012r2/build.gradle

```diff
@@ -1,11 +1,12 @@
+import org.elasticsearch.gradle.test.GradleDistroTestTask
 
 String boxId = project.properties.get('vagrant.windows-2012r2.id')
 if (boxId != null) {
   vagrant {
     hostEnv 'VAGRANT_WINDOWS_2012R2_BOX', boxId
   }
 } else {
-  tasks.named('distroTest').configure {
+  tasks.withType(GradleDistroTestTask).configureEach {
```

Comment:
We should probably include a comment here as to "why" we are doing this. Even if it's temporary.

Reply:
This isn't temporary, it's just disable the distribution tests for windows when we don't have a windows image to test with. I've added a comment.